CMakeLists.txt

```diff
@@ -650,7 +655,8 @@ message("\n")
 if(configure_warnings)
     message("  ******\n")
     foreach(warning IN LISTS configure_warnings)
-      message(WARNING "${warning}")
+      message(CONFIGURE_LOG "${warning}") # Log to the cmake-configure-log file before a potentially fatal message.
```

Comment:
`message(CONFIGURE_LOG ...)` was [added](https://cmake.org/cmake/help/latest/command/message.html#configure-log) in CMake version 3.26. Our minimum required one is 3.22.

Reply:
Thanks, fixed.